src/main/java/org/mitre/synthea/export/FhirStu3.java

```diff
@@ -826,12 +826,11 @@ private static BundleEntryComponent encounterClaim(BundleEntryComponent personEn
   private static BundleEntryComponent condition(BundleEntryComponent personEntry, Bundle bundle,
       BundleEntryComponent encounterEntry, HealthRecord.Entry condition) {
     Condition conditionResource = new Condition();
-
     conditionResource.setSubject(new Reference(personEntry.getFullUrl()));
     conditionResource.setContext(new Reference(encounterEntry.getFullUrl()));
 
     Code code = condition.codes.get(0);
-    conditionResource.setCode(mapCodeToCodeableConcept(code, SNOMED_URI));
+    conditionResource.setCode(mapCodeToCodeableConcept(code, code.system));
```

Comment:
The codeLookup from the issue below could be used to map the system to a URI.  Would this just be loading the file in the exporter and using it in `mapCodeToCodeableConcept`?

Reply:
Also do you think it would be better to just use the lookup table to translate the systems when they're received by `Terminology`, or should I transfer the lookup function to the exporter classes, and make the switch to URI only right before it gets exported?